packages/cli/src/args.ts

```diff
@@ -0,0 +1,80 @@
+import chalk from 'chalk';
+import logo from './util/output/logo';
+import { getPkgName } from './util/pkg-name';
+
+export const help = () => `
+  ${chalk.bold(`${logo} ${getPkgName()}`)} [options] <command | path>
+
+  ${chalk.dim('Commands:')}
+
+    ${chalk.dim('Basic')}
+
+      deploy               [path]      Performs a deployment ${chalk.bold(
+        '(default)'
+      )}
+      dev                              Start a local development server
+      env                              Manages the Environment Variables for your current Project
+      git                              Manage Git provider repository for your current Project
+      help                 [cmd]       Displays complete help for [cmd]
+      init                 [example]   Initialize an example project
+      inspect              [id]        Displays information related to a deployment
+      link                 [path]      Link local directory to a Vercel Project
+      ls | list            [app]       Lists deployments
+      login                [email]     Logs into your account or creates a new one
+      logout                           Logs out of your account
+      pull                 [path]      Pull your Project Settings from the cloud
+      rollback             [url|id]    Quickly revert back to a previous deployment [beta]
+      switch               [scope]     Switches between teams and your personal account
+
+    ${chalk.dim('Advanced')}
+
+      alias                [cmd]       Manages your domain aliases
+      bisect                           Use binary search to find the deployment that introduced a bug
+      certs                [cmd]       Manages your SSL certificates
+      dns                  [name]      Manages your DNS records
+      domains              [name]      Manages your domain names
+      logs                 [url]       Displays the logs for a deployment
+      projects                         Manages your Projects
+      rm | remove          [id]        Removes a deployment
+      secrets              [name]      Manages your global Secrets, for use in Environment Variables
+      teams                            Manages your teams
+      whoami                           Shows the username of the currently logged in user
+
+  ${chalk.dim('Options:')}
```

Comment:
This is still confusing to me. These are technically global options which is what our docs call them: https://vercel.com/docs/cli/global-options.

I think if this section was renamed to "Global Options:", then add above it a "Deploy Options:" (or simply "Options:") with the `vc deploy` options, that would provide a better user experience.

Reply:
How about this? https://github.com/vercel/vercel/pull/9917

Reply:
Love it!